corehq/blobs/models.py

```diff
@@ -128,17 +127,3 @@ class DeletedBlobMeta(PartitionedModel, Model):
 class BlobMigrationState(Model):
     slug = CharField(max_length=20, unique=True)
     timestamp = DateTimeField(auto_now=True)
-
-
-class BlobExpiration(Model):
```

Comment:
This is not strictly "safe" to remove at the same time that the task depends on it. However even if some old code runs on this model the failure mode will be that the task to delete blobs will fail and will be retried in the next 12 hours.